Tests/LibWeb/TestMimeSniff.cpp

```diff
@@ -28,10 +29,10 @@ TEST_CASE(determine_computed_mime_type_given_no_sniff_is_set)
 
 TEST_CASE(determine_computed_mime_type_given_no_sniff_is_unset)
 {
-    auto supplied_type = MUST(Web::MimeSniff::MimeType::create("text"_string, "html"_string));
```

Comment:
I don't remember what this tested, but it looks like it might have been the removed resource feed branch seeing that it is was failing. It now just tests to see if we can reach step 9 of the algo.